deploy/application/java/java-maven.md

```diff
@@ -34,8 +34,8 @@ Maven is essentially a project management and comprehension tool and as such pro
 ## Configure your Java application
 
 ### About Cargo
-To run your app, you can, for example, use plugins like cargo
-(<a href="https://codehaus-cargo.GitHub.io/cargo/Maven2+plugin.html">Find it here</a>).
+
+To run your app, you can, for example, use plugins like cargo ([Find it here](https://codehaus-cargo.GitHub.io/cargo/Maven2+plugin.html)).
```

Comment:
This links seems to now redirect to https://codehaus-cargo.github.io/cargo/Maven+3+Plugin.html and since it's not a clean redirection (it's using JavaScript), we better update it